src/components/Review.jsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import { useState, useEffect } from 'react'
+import { getReviews } from '../utils/utils';
+import { Link, useParams } from 'react-router-dom';
+import { getReviewById, getComments } from '../utils/utils';
+
+
+
+
+function Review() {
+    const [loading, setLoading] = useState(true);
+    const { review_id } = useParams();
+    const [comments, setComments] = useState(
+        [
+            {
+                "comment_id": 41,
+                "body": "Ex id ipsum dolore non cillum anim sint duis nisi anim deserunt nisi minim. Fugiat sint et proident ex do consequat est. Nisi minim laboris mollit cupidatat.",
+                "review_id": 18,
+                "author": "jessjelly",
+                "votes": 9,
+                "created_at": "2021-03-27T14:15:31.110Z"
+                }
+        ]
+
+    )
+ function deleteComments(id){
+}
+
+    
+    const [review, setReview] = useState(
+        [{
+            category: "strategy",
+            comment_count: "4",
+            created_at: "1970-01-10T02:56:38.400Z",
+            designer: "Emerson Matsuuchi",
+            owner: "tickle122",
+            review_body: "This game reminds me of the stress-free environment described in a song sung by a crab in the famous film about a mermaid. Plenty of coral collecting, and reef building to keep you entertained ",
+            review_id: 18,
+            review_img_url: "https://images.pexels.com/photos/6333894/pexels-photo-6333894.jpeg?w=700&h=700",
+            title: "Reef",
+            votes: 6
+        }]
+    );
+
+    useEffect(()=>{
+        setLoading(true);
+        getReviewById(review_id)
+        .then((data) => {
+            setReview(data)
+            getComments(review_id)
+            .then((data) => {
+                // console.log(data);
+                setComments(data)
+      
+            })
+        })
+        .then(() => {
+            setLoading(false)
+        })
+    }, [])
+
+
+    
+    
+
+
+    
+    useEffect(() => {
+        if(loading === true){
+            document.getElementById('loadingScreen').style.display = 'block';
+        } else {
+            document.getElementById('loadingScreen').style.display = 'none';
+        }
+    }, [loading])
```

Comment:
I see what you've done here, however, it's a lot cleaner and efficient to have the loading message appear by conditional rendering rather than in a useEffect. You have 2 useEffects on this page which will re-render the whole review component. If we use conditional rendering below, there is no need for this useEffect. Give me a shout if you need this to be clarified 🙂